src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,16 +1,127 @@
-import { Component } from 'react';
-
-type Props = {};
-type State = {};
-
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
-
-  render() {
-    return (
-      <form>
-        Put the form here
-      </form>
-    );
-  }
-}
+import React, { useState } from 'react';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({
+  addMovie,
+}) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+
+  const newMovie: Movie = {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  };
+
+  const handleSubmit = () => {
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
+  };
+
+  return (
+    <form
+      className="NewMovie"
+      onSubmit={(event) => {
+        event.preventDefault();
+        addMovie(newMovie);
+        handleSubmit();
+      }}
+    >
+      <div>
+        <label className="label">
+          <span>{'Title '}</span>
```

Comment:
redundant brackets, it's just a text, fix it everywhere
```suggestion
          <span>Title</span>
```